### PR TITLE
use sample.launch as default demo program

### LINKED ISSUE #22: why we have both sample.launch and run.launch
- sample.launch publishes /base -> /sensor tf
- sample.launch includes run.launch
- run.launch start rviz
- rviz's fixed frame is /base
- doc using run.launch

I guess sample.launch is designed for sample program running driver and rviz, where as run.launch is intend to use within other project, which already publishing tf and rviz.
So may be we need to move rviz node from run.launch to sample.launch and change doc to use sample.launch

### PR DESCRIPTION
**Pull request by k-okada**
also renamed run.launch -> driver.launch, not sure if this breaks someone useig this...

This Closes #22

- fix documents
- rename run -> driver not to confuse
- use sample.launch as sample program, run.launch is just to run driver nodes

### CODE REVIEW

**Comment by 130s** [comment]
LGTM except one this: I still want RViz node to be in a root launch file knowing that end users don't even want to bother writing their own launch files.

**Comment by k-okada** [comment]
This is the use case;
1. If you just want to test the sensor alone, use sample.launch, that's runs driver, tf and Rviz. You can check if the sensor is working.
2. If you already integrate the sensor within your robot, you already have your own launch files that publish sensor tf frames and you already put your rviz node with in your own launch file, so you just include driver.launch

So If we put rviz node within driver.launch, you will start two rviz not one comes from driver.launch and another comes from your own launch file for your robot.

**Comment by 130s** [comment]
I agree with the usecase.

Only thing I care is that end users may not even want to write launch files as much as possible. Tasks like e.g passing .rviz file are a simple for experienced users, but not for majority of users. That's why I included RViz node in run/driver.launch (, which is made as an option, not mandatory. Please see https://github.com/k-okada/dynpick_driver/issues/1).

**Comment by k-okada** [comment]
what do you mean by "end users", if you're running robot, you do not have
to run rviz, your robot launch file, like
nextage_ros_bridge_simulation.launch already runs rviz, so you'll create
nextage_ros_bridge_simulation_force_sensor.launch that includes both
nextage_ros_bridge_simulaiton.launch and driver.launch.
If you're testing sensros, use sample.launch, which runs rviz.


--
◉ Kei Okada

File: doc/index.rst [hunk 1]
```diff
@@ -8,8 +8,8 @@ ROS driver for `Wacoh-tech force sensor <http://www.wacoh-tech.com/en/products/d
 Prerequisite
 ==============
 
- * (Only tested on) Ubuntu 12.04 and 14.04 64bit.
- * ROS Hydro or Indigo needs to be installed.
+ * (Only tested on) Ubuntu 14.04 64bit.
+ * ROS Indigo needs to be installed.
```
Comment (130s) [inline doc/index.rst]: I think we don't need to remove already-tested platform from the document.

**Comment by 130s** [comment]
> what do you mean by "end users", 

プログラムを実行する人を指したつもりです．

> if you're running robot, you do not have to run rviz, your robot launch file, like nextage_ros_bridge_simulation.launch already runs rviz, so you'll create extage_ros_bridge_simulation_force_sensor.launch that includes both extage_ros_bridge_simulaiton.launch and driver.launch. If you're testing sensros, use sample.launch, which runs rviz.

That example makes sense to me. 
In RViz currently there's no way for the dependent packages (e.g. robot pkg like nextage_ros_bridge) to incorporate `.rviz` file that's specific to the driver (AFAIK), so it's dependent packages that have to prepare .rviz for the driver pkg.

+1. Merging.

**Pull request merged**